extensions/arc/src/common/utils.ts

```diff
@@ -218,6 +218,61 @@ export function createCredentialId(controllerId: string, resourceType: string, i
 }
 
 /**
+ * Calculates the gibibyte (GiB) conversion of a quantity that could currently be represented by a range
+ * of SI suffices (E, P, T, G, M, K, m) or their power-of-two equivalents (Ei, Pi, Ti, Gi, Mi, Ki)
+ * @param value The string of a quantity to be converted
+ * @returns String of GiB conversion
+ */
+export function convertToGibibyteString(value: string): string {
```

Comment:
I think this function should `throw new Error(`${value} is not a valid Kubernetes resource quantity`)` if `value` doesn't have the format we expect (a number optionally followed by one of the known units).